__tests__/integrations/slack.ts

```diff
@@ -150,7 +150,7 @@ describe('GET /integrations/slack/auth/callback', () => {
     ).expect(307);
 
     expect(body).toEqual({});
-    expect(headers.location).toBe('http://localhost:5002/squads/test');
+    expect(headers.location).toContain('http://localhost:5002/squads/test');
```

Comment:
We can parse it as URL and then check pathname + `iid` query, this feels like it can miss something with `toContain`

Reply:
Yeah happy with that :)